examples/pylab_examples/scatter_demo2.py

```diff
@@ -12,7 +12,7 @@
 datafile = cbook.get_sample_data('goog.npy')
 try:
     # Python3 cannot load python2 .npy files with datetime(object) arrays
-    # unless the encoding is set to bytes. Hovever this option was
+    # unless the encoding is set to bytes. However this option was
     # not added until numpy 1.10 so this example will only work with
     # python 2 or with numpy 1.10 and later
     price_data = np.load(datafile, encoding='bytes').view(np.recarray)
```

Comment:
np.datetime64 has been a mess from the start.  It is marginally usable now that we require numpy 1.7, so we do need to add support as best we can.  Maybe someone will want to take that on for 2.1.